<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.8.11">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Spencer S. Hong">
<meta name="dcterms.date" content="2025-01-01">

<title>18&nbsp; Beyond Traditional Views of Science Funding – AI-NEPI Conference Proceedings - Enhanced Edition</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<link href="./chapter_ai-nepi_021.html" rel="next">
<link href="./chapter_ai-nepi_019.html" rel="prev">
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/axe/axe-check.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-09b140d2d032adf2aedb8b099be3ee13.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-350fb9e808f7eb2950c9598fb3f8c4a0.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "sidebar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "start",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-sidebar floating quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" role="button" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
        <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="./chapter_ai-nepi_020.html"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Beyond Traditional Views of Science Funding</span></a></li></ol></nav>
        <a class="flex-grow-1" role="navigation" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
        </a>
      <button type="button" class="btn quarto-search-button" aria-label="Search" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal quarto-sidebar-collapse-item sidebar-navigation floating overflow-auto">
    <div class="pt-lg-2 mt-2 text-left sidebar-header">
    <div class="sidebar-title mb-0 py-0">
      <a href="./">AI-NEPI Conference Proceedings - Enhanced Edition</a> 
    </div>
      </div>
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">1</span>&nbsp; <span class="chapter-title">AI-NEPI Conference Proceedings - Enhanced Edition</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ai-nepi_001_chapter.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">2</span>&nbsp; <span class="chapter-title">Large Language Models for the History, Philosophy and Sociology of Science (Workshop)</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ai-nepi_003_chapter.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">3</span>&nbsp; <span class="chapter-title">The Transformer’s Legacy: Understanding Large Language Models and their Application in HPSS Research</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ai-nepi_004_chapter.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">4</span>&nbsp; <span class="chapter-title">The Workflow and Utility of OpenAlex Mapper</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ai-nepi_005_chapter.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">5</span>&nbsp; <span class="chapter-title">Genre Classification for Historical Medical Periodicals</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ai-nepi_006_chapter.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">6</span>&nbsp; <span class="chapter-title">Towards Computational HPSS: Tracing the Influence of the Ancient Wisdom Tradition on Early Modern Science using the LLM-powered Semantic Matching Tool of the VERITRACE project</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ai-nepi_007_chapter.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">7</span>&nbsp; <span class="chapter-title">Interpretability for LLMs: Transparency, Applications and Scientific Insights in the Humanities</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./ai-nepi_008_chapter.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">8</span>&nbsp; <span class="chapter-title">Modeling Science: LLM for the History, Philosophy and Sociology of Science</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_009.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">9</span>&nbsp; <span class="chapter-title">Leveraging Large Language Models to Assess Biases in Sustainable Development Goal Classifications Across Major Bibliometric Databases</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_010.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">10</span>&nbsp; <span class="chapter-title">Large Language Models for Footnote Parsing in Law and Humanities Scholarship</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_011.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">11</span>&nbsp; <span class="chapter-title">Science Dynamics and AI</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_012.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">12</span>&nbsp; <span class="chapter-title">Retrieval Augmented Generation (RAG) in Philosophical Research: Applications and Methodological Challenges</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_015.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">13</span>&nbsp; <span class="chapter-title">Quantum Gravity and Plural Pursuit: A Multi-Scale Investigation</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_016.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">14</span>&nbsp; <span class="chapter-title">Titles, Abstracts, or Full-Texts? A Comparative Study of <em>LDA</em> and <em>BERTopic</em> Performance across Text Levels</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_017.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">15</span>&nbsp; <span class="chapter-title">Time-Aware Language Models</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_018.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">16</span>&nbsp; <span class="chapter-title">Leverage Large Language Models for Metadata Enrichment and Diachronic Analysis of Chemical Knowledge in Historical Scientific Texts</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_019.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Modelling Context and its Interplay in Language Variation and Change: A Pilot Study on the Chemical Revolution</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_020.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Beyond Traditional Views of Science Funding</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./chapter_ai-nepi_021.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">From Source to Structure: Extracting Knowledge Graphs with LLMs</span></span></a>
  </div>
</li>
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="./references.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text"><span class="chapter-number">20</span>&nbsp; <span class="chapter-title">References</span></span></a>
  </div>
</li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" class="quarto-sidebar-collapse-item" data-bs-toggle="collapse" data-bs-target=".quarto-sidebar-collapse-item"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#overview" id="toc-overview" class="nav-link active" data-scroll-target="#overview">Overview</a></li>
  <li><a href="#limitations-of-current-science-funding-analysis" id="toc-limitations-of-current-science-funding-analysis" class="nav-link" data-scroll-target="#limitations-of-current-science-funding-analysis"><span class="header-section-number">18.1</span> Limitations of Current Science Funding Analysis</a></li>
  <li><a href="#the-human-genome-project-a-paradigm-of-big-science" id="toc-the-human-genome-project-a-paradigm-of-big-science" class="nav-link" data-scroll-target="#the-human-genome-project-a-paradigm-of-big-science"><span class="header-section-number">18.2</span> The Human Genome Project: A Paradigm of ‘Big Science’</a></li>
  <li><a href="#nhgri-an-innovative-funding-agency" id="toc-nhgri-an-innovative-funding-agency" class="nav-link" data-scroll-target="#nhgri-an-innovative-funding-agency"><span class="header-section-number">18.3</span> NHGRI: An Innovative Funding Agency</a></li>
  <li><a href="#the-nhgri-archive-a-rich-and-complex-source" id="toc-the-nhgri-archive-a-rich-and-complex-source" class="nav-link" data-scroll-target="#the-nhgri-archive-a-rich-and-complex-source"><span class="header-section-number">18.4</span> The NHGRI Archive: A Rich and Complex Source</a></li>
  <li><a href="#internal-archives-versus-public-data" id="toc-internal-archives-versus-public-data" class="nav-link" data-scroll-target="#internal-archives-versus-public-data"><span class="header-section-number">18.5</span> Internal Archives Versus Public Data</a></li>
  <li><a href="#computational-methodologies-for-archive-processing" id="toc-computational-methodologies-for-archive-processing" class="nav-link" data-scroll-target="#computational-methodologies-for-archive-processing"><span class="header-section-number">18.6</span> Computational Methodologies for Archive Processing</a></li>
  <li><a href="#entity-and-pii-recognition" id="toc-entity-and-pii-recognition" class="nav-link" data-scroll-target="#entity-and-pii-recognition"><span class="header-section-number">18.7</span> Entity and PII Recognition</a></li>
  <li><a href="#reconstructing-the-nhgri-correspondence-network" id="toc-reconstructing-the-nhgri-correspondence-network" class="nav-link" data-scroll-target="#reconstructing-the-nhgri-correspondence-network"><span class="header-section-number">18.8</span> Reconstructing the NHGRI Correspondence Network</a></li>
  <li><a href="#information-brokerage-and-leadership-dynamics" id="toc-information-brokerage-and-leadership-dynamics" class="nav-link" data-scroll-target="#information-brokerage-and-leadership-dynamics"><span class="header-section-number">18.9</span> Information Brokerage and Leadership Dynamics</a></li>
  <li><a href="#modelling-funding-decisions-for-genome-sequencing" id="toc-modelling-funding-decisions-for-genome-sequencing" class="nav-link" data-scroll-target="#modelling-funding-decisions-for-genome-sequencing"><span class="header-section-number">18.10</span> Modelling Funding Decisions for Genome Sequencing</a></li>
  <li><a href="#interpreting-funding-decisions-the-matthew-effect" id="toc-interpreting-funding-decisions-the-matthew-effect" class="nav-link" data-scroll-target="#interpreting-funding-decisions-the-matthew-effect"><span class="header-section-number">18.11</span> Interpreting Funding Decisions: The Matthew Effect</a></li>
  <li><a href="#broader-applications-from-archives-to-knowledge" id="toc-broader-applications-from-archives-to-knowledge" class="nav-link" data-scroll-target="#broader-applications-from-archives-to-knowledge"><span class="header-section-number">18.12</span> Broader Applications: From Archives to Knowledge</a></li>
  <li><a href="#preservation-collaboration-and-nhgris-significance" id="toc-preservation-collaboration-and-nhgris-significance" class="nav-link" data-scroll-target="#preservation-collaboration-and-nhgris-significance"><span class="header-section-number">18.13</span> Preservation, Collaboration, and NHGRI’s Significance</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title"><span class="chapter-number">18</span>&nbsp; <span class="chapter-title">Beyond Traditional Views of Science Funding</span></h1>
</div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Author</div>
  <div class="quarto-title-meta-heading">Affiliation</div>
  
    <div class="quarto-title-meta-contents">
    <p class="author">Spencer S. Hong </p>
  </div>
  <div class="quarto-title-meta-contents">
        <p class="affiliation">
            AI-NEPI Conference Participant
          </p>
      </div>
  </div>

<div class="quarto-title-meta">

      
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">January 1, 2025</p>
    </div>
  </div>
  
    
  </div>
  


</header>


<section id="overview" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="overview">Overview</h2>
<p>Researchers meticulously investigate the processes and impacts of science funding, moving beyond conventional bibliometric analyses of publications and grants. This inquiry focuses upon the National Human Genome Research Institute (NHGRI), renowned for its pioneering contributions to biology, particularly through the Human Genome Project. An interdisciplinary team, encompassing historians, physicists, ethicists, computer scientists, and former NHGRI leadership, analyses an extensive internal archive. This repository contains over two million pages of documents from the NHGRI, including meeting notes, handwritten correspondence, presentations, spreadsheets, and emails. Growing by 5% annually through continuous digitisation, this archive offers unparalleled insight into the internal workings of a major funding agency.</p>
<p>Methodologically, the project employs advanced computational techniques to process and analyse these born-physical and born-digital artefacts. These techniques include developing a custom-built handwriting removal model using a <em>U-Net</em> architecture. This process enhances the accuracy of Optical Character Recognition (OCR) for printed text whilst enabling a distinct processing pipeline solely for handwriting recognition. Multimodal models, harnessing visual, textual, and structural modalities, facilitate sophisticated tasks such as entity extraction and synthetic data generation to train classifiers. Robust systems for entity and Personally Identifiable Information (PII) recognition meticulously mask and disambiguate sensitive data, including names, organisations, email addresses, locations, and identification numbers, achieving high F1 scores with relatively small fine-tuned datasets.</p>
<p>A pivotal aspect of the research involved reconstructing and analysing the NHGRI’s internal and external email correspondence network. By extracting entities from thousands of scanned paper emails and linking them, researchers successfully recreated an intricate web of communication from the Human Genome Project era. This reconstructed network encompasses 62,511 distinct email conversations originating from 5,414 individual scanned emails. Network analysis, including <em>stochastic block modelling</em> and <em>brokerage role analysis</em>, uncovered leadership structures, such as an informal ‘Kitchen Cabinet’ within the International HapMap Project. It also revealed patterns of information flow, suggesting NHGRI leadership frequently adopted a consultative, rather than gatekeeping, posture.</p>
<p>Furthermore, the research explored funding decisions by computationally modelling the factors influencing organism selection for genome sequencing post-Human Genome Project. This model integrated biological features (e.g., genome size, distance to model organisms), project characteristics (e.g., team size, submission history, gender equity, internal proposal origin), applicant reputational metrics (e.g., H-index, community size, network centrality), and linguistic characteristics of proposals (e.g., argumentation, repetitiveness). The findings suggest that all these feature categories collectively predicted funding success, with reputational factors like H-index and community size demonstrating a Matthew Effect.</p>
<p>The project’s broader objective centres on rendering such born-physical archives Findable, Accessible, Interoperable, and Reusable (FAIR). This involves applying these methodologies to other extensive archives, including federal court records and seismological data. Crucially, this endeavour underscores the imperative of preserving these rich historical data sources and developing sophisticated tools to extract knowledge, inform policy, enhance data accessibility, and address pressing scientific questions. The consortium actively seeks testers, partners, and users for its ‘Born Physical, Studied Digitally’ initiative, which receives support from the NIH, <em>NVIDIA</em>, and the NSF.</p>
</section>
<section id="limitations-of-current-science-funding-analysis" class="level2" data-number="18.1">
<h2 data-number="18.1" class="anchored" data-anchor-id="limitations-of-current-science-funding-analysis"><span class="header-section-number">18.1</span> Limitations of Current Science Funding Analysis</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_01.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 01</figcaption>
</figure>
</div>
<p>State-sponsored research has profoundly shaped the scientific landscape since the Second World War. It operates under a social contract where public funds support research endeavours, expecting them to yield societal benefits such as informed policy, clinical advancements, and new technologies. Scholars in the science of science predominantly study this enterprise by examining publicly accessible data, primarily scientific publications and grant awards. Indeed, analyses of these sources, including bibliometrics, have offered valuable insights into diverse aspects of science, including its long-term impact, the evolution of team sizes, the emergence of interdisciplinary fields, and the career trajectories of scientists.</p>
<p>Nevertheless, relying solely on the scientific article presents an incomplete, even skewed, representation of the complex scientific process. To assume that bibliometrics fully encapsulates the essence of science constitutes an oversimplification. A deeper understanding necessitates investigating the processes that precede publication, moving beyond the flawed picture painted by articles alone. Such an approach could illuminate critical questions: Does scientific inquiry shape funding priorities, or do funding agendas dictate the direction of science? Within the innovation pipeline, from initial ideation to eventual long-term impact, where do innovations flourish, diffuse across domains, or ultimately falter?</p>
<p>Notably, the focus on published articles means that failed projects, which could offer significant learning opportunities, often remain unexamined. Beyond direct financial support, funders may also contribute through public data provision, community engagement initiatives, technology development, and cooperative agreements. All these factors influence both the creation of knowledge and the scholars involved. A dynamic interplay also exists between grant funding and technological development.</p>
</section>
<section id="the-human-genome-project-a-paradigm-of-big-science" class="level2" data-number="18.2">
<h2 data-number="18.2" class="anchored" data-anchor-id="the-human-genome-project-a-paradigm-of-big-science"><span class="header-section-number">18.2</span> The Human Genome Project: A Paradigm of ‘Big Science’</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_04.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 04</figcaption>
</figure>
</div>
<p>The Human Genome Project (HGP) stands as a landmark example of ‘big science’ in biology, analogous to large-scale projects in fields like particle physics. This colossal undertaking brought together tens of countries and thousands of researchers with the shared goal of sequencing the entire human genome. Its significance extends across multiple dimensions. Firstly, the HGP captured public interest to an extent previously unseen for a biological research programme, shifting focus from laboratory-based studies of organisms like <em>Drosophila</em> and <em>C. elegans</em> to a grand human-centric endeavour. Secondly, its impact resonates profoundly today; a vast majority of modern biological research, particularly omics methodologies, would be unfeasible without the reference genome it produced. Indeed, the HGP effectively established genomics as a distinct scientific discipline.</p>
<p>Furthermore, the project pioneered new data-sharing practices, now widely adopted, and forged a powerful synergy between computational science and biology. Two principal organisations spearheaded this international effort: the Wellcome Trust in the United Kingdom and, crucially for this study, the National Human Genome Research Institute (NHGRI), which served as the HGP division within the US National Institutes of Health (NIH). Dr Francis Collins, then Director of the NHGRI and later Director of the NIH, played a pivotal leadership role.</p>
</section>
<section id="nhgri-an-innovative-funding-agency" class="level2" data-number="18.3">
<h2 data-number="18.3" class="anchored" data-anchor-id="nhgri-an-innovative-funding-agency"><span class="header-section-number">18.3</span> NHGRI: An Innovative Funding Agency</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_04.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 04</figcaption>
</figure>
</div>
<p>Analysis reveals the National Human Genome Research Institute (NHGRI) as a particularly innovative funding body within the National Institutes of Health (NIH). Several bibliometric indicators support this assessment when comparing NHGRI to other NIH institutes. For instance, NHGRI-funded research accounts for a larger share of manuscripts in the top 5% most cited publications. Moreover, its output demonstrates substantial long-term citation impact (measured after ten years) and generates significant citations from patents, indicating translation into clinical applications. The research funded by NHGRI also exhibits high ‘disruption’ scores, suggesting it often pioneers new directions.</p>
<p>Whilst these metrics establish NHGRI’s innovative capacity, the underlying reasons for this success remain less understood. Consequently, an interdisciplinary research team has assembled to investigate the processes and practices that enable NHGRI to lead innovation. This team comprises experts from diverse fields, including history, physics, ethics, and computer science, and notably includes Dr Francis Collins, a former Director of both NHGRI and NIH. Their collective aim is to unravel the factors contributing to the rise of genomics, identify potential failure points and innovation spillovers, and understand the collaborative mechanisms between funding agencies and academic scientists that foster scientific advancement.</p>
</section>
<section id="the-nhgri-archive-a-rich-and-complex-source" class="level2" data-number="18.4">
<h2 data-number="18.4" class="anchored" data-anchor-id="the-nhgri-archive-a-rich-and-complex-source"><span class="header-section-number">18.4</span> The NHGRI Archive: A Rich and Complex Source</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_07.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 07</figcaption>
</figure>
</div>
<p>Owing to the recognised historical importance of the Human Genome Project, the NHGRI meticulously preserved a substantial collection of its internal documentation, spanning from the 1980s and 1990s into subsequent years. This internal archive constitutes a rich repository, containing diverse materials such as the daily meeting notes of scientists coordinating the genome project, handwritten annotations from correspondence, conference agendas, formal presentations, detailed spreadsheets, and newspaper clippings chronicling the period. Additionally, it includes various internal forms, research proposals, and extensive email communications.</p>
<p>The sheer volume of this collection, currently exceeding two million pages and expanding by approximately 5% each year due to continuous digitisation, presents a formidable challenge. Effectively analysing such a vast and complex born-physical and born-digital artefact at scale necessitates innovative approaches, forming the core of the research team’s methodological development.</p>
</section>
<section id="internal-archives-versus-public-data" class="level2" data-number="18.5">
<h2 data-number="18.5" class="anchored" data-anchor-id="internal-archives-versus-public-data"><span class="header-section-number">18.5</span> Internal Archives Versus Public Data</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_09.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 09</figcaption>
</figure>
</div>
<p>The internal documents housed within the NHGRI archive possess characteristics and content fundamentally distinct from publicly accessible information, such as Requests for Applications (RFAs) and peer-reviewed publications. Whilst scholars can readily access RFAs and publications through databases like <em>PubMed</em> or <em>NIH RePORTER</em>, the internal records offer a different perspective. Visualisations, such as <em>t-SNE</em> plots, demonstrate that these internal materials form distinct clusters, separate from the clusters representing public RFAs and publications.</p>
<p>These internal records provide detailed accounts of numerous large-scale genomic projects initiated and funded by NHGRI. Examples include the Ethical, Legal, and Social Implications (ELSI) Program’s LSAC, <em>modENCODE</em>, <em>eMERGE</em>, <em>ENCODE</em>, the foundational Human Genome Project itself, <em>PAGE</em>, the International HapMap Project, <em>H3Africa</em>, and the NHGRI-EBI GWAS Catalog. Many of these initiatives represented substantial investments, often amounting to tens or hundreds of millions of dollars, and mobilised thousands of researchers globally. Their collective purpose was to develop crucial resources for the genomics community, thereby catalysing the advancement of the entire field.</p>
</section>
<section id="computational-methodologies-for-archive-processing" class="level2" data-number="18.6">
<h2 data-number="18.6" class="anchored" data-anchor-id="computational-methodologies-for-archive-processing"><span class="header-section-number">18.6</span> Computational Methodologies for Archive Processing</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_10.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 10</figcaption>
</figure>
</div>
<p>Researchers developed specialised computational methods to manage the born-physical archive, a significant portion of which contains handwritten material. The use of AI for handwriting analysis presents not only technical hurdles but also ethical considerations regarding the unknown nature of handwritten content. To address this, the team trained a custom-built handwriting model, employing a <em>U-Net</em> architecture, specifically to identify and remove handwritten portions from documents. This process offers a dual benefit: it enhances the accuracy of Optical Character Recognition (OCR) on the remaining printed text and simultaneously enables the creation of a distinct processing pipeline dedicated solely to handwriting recognition.</p>
<p>Beyond handwriting, the project leverages advances in multimodal models, drawing from the document intelligence research community. These models ingeniously combine visual information (the document image), textual content, and layout structure. Such an integrated approach supports diverse tasks, including sophisticated entity extraction. Furthermore, it facilitates the generation of synthetic documents, which serve as valuable training data for developing and refining new classification algorithms, thereby improving the system’s analytical capabilities over time.</p>
</section>
<section id="entity-and-pii-recognition" class="level2" data-number="18.7">
<h2 data-number="18.7" class="anchored" data-anchor-id="entity-and-pii-recognition"><span class="header-section-number">18.7</span> Entity and PII Recognition</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_12.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 12</figcaption>
</figure>
</div>
<p>A critical aspect of processing the NHGRI archive involves the meticulous handling of sensitive information. The documents contain genuine Personally Identifiable Information (PII), including details of real individuals, credit card numbers, and social security numbers, pertaining to people some of whom remain active in governmental and academic roles today. Consequently, the researchers implemented robust methods for entity and PII recognition. Their system effectively identifies, masks, and disambiguates such sensitive data throughout the vast collection.</p>
<p>The performance of these recognition models proves strong, as evidenced by F1 scores that improve with increased fine-tuning data for various entity types. These include persons (PERSON), organisations (ORG), email addresses (EMAIL), locations (LOC), and identification numbers (IDNUM). This careful approach ensures privacy and ethical compliance whilst enabling scholarly analysis of the archive’s content.</p>
</section>
<section id="reconstructing-the-nhgri-correspondence-network" class="level2" data-number="18.8">
<h2 data-number="18.8" class="anchored" data-anchor-id="reconstructing-the-nhgri-correspondence-network"><span class="header-section-number">18.8</span> Reconstructing the NHGRI Correspondence Network</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_13.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 13</figcaption>
</figure>
</div>
<p>Researchers conducted a detailed case study involving the reconstruction of email networks from the NHGRI archive. By extracting entities from thousands of scanned paper copies of emails and linking them, they successfully recreated the intricate web of correspondence that occurred during the Human Genome Project era. This reconstructed network encompasses 62,511 distinct email conversations originating from 5,414 individual scanned emails. Each node in the visualised network typically represents an individual involved in these communications, with affiliations to NIH (National Institutes of Health) or external entities (such as other funding agencies, companies, or universities) clearly delineated.</p>
<p>Applying network analysis techniques, such as <em>stochastic block modelling</em> for community detection, yielded significant insights, particularly concerning the coordination of large-scale initiatives like the International HapMap Project. The HapMap Project, a major genomics endeavour following the HGP, focused on human genetic variation and laid the groundwork for genome-wide association studies (GWAS). Managing such a complex project, involving numerous universities and agencies, relied on formal structures like a steering committee. However, the computational analysis, performed in an unsupervised manner, revealed a hitherto undocumented informal leadership circle, dubbed the ‘Kitchen Cabinet’. This group, referencing a term from the Nixonian political era, apparently convened prior to official steering committee meetings, likely to address potential issues proactively and ensure smoother formal proceedings.</p>
</section>
<section id="information-brokerage-and-leadership-dynamics" class="level2" data-number="18.9">
<h2 data-number="18.9" class="anchored" data-anchor-id="information-brokerage-and-leadership-dynamics"><span class="header-section-number">18.9</span> Information Brokerage and Leadership Dynamics</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_16.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 16</figcaption>
</figure>
</div>
<p>Further investigation into the NHGRI’s operational dynamics focused on comparing the communication patterns of the informal ‘Kitchen Cabinet’ with those of the formal Steering Committee. Utilising <em>brokerage role analysis</em>, which assesses how individuals or groups mediate information flow within a network, researchers identified distinct behavioural patterns. This analysis categorises nodes based on their interaction styles, such as ‘consultant’ (receiving information and disseminating it back within their own group) or ‘gatekeeper’ (receiving information but not sharing it back with the originating group).</p>
<p>The findings indicate that the ‘Kitchen Cabinet’ primarily functioned in a consultant capacity, a pattern that distinguished it from other formal leadership structures active at the time. Notably, individuals like Francis Collins appeared to play significant consultant roles within this informal group. This suggests a leadership style at NHGRI that favoured consultation and open information exchange rather than restrictive gatekeeping, potentially contributing to the agency’s collaborative successes.</p>
</section>
<section id="modelling-funding-decisions-for-genome-sequencing" class="level2" data-number="18.10">
<h2 data-number="18.10" class="anchored" data-anchor-id="modelling-funding-decisions-for-genome-sequencing"><span class="header-section-number">18.10</span> Modelling Funding Decisions for Genome Sequencing</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_13.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 13</figcaption>
</figure>
</div>
<p>The research extended to analysing the funding agency’s decision-making processes through portfolio analysis, specifically modelling the choices made when selecting non-human organisms for genome sequencing following the completion of the Human Genome Project. Funding agencies like NHGRI faced complex decisions in allocating resources amongst numerous proposals from different organismal research communities, each advocating for their chosen species (e.g., various primates). To understand these decisions, scientists developed a machine learning model designed to recapitulate the actual funding outcomes.</p>
<p>This computational model incorporated a diverse array of features. Biological characteristics, such as the proposed organism’s genome size and its evolutionary distance to already sequenced model organisms, demonstrated predictive value (Area Under Curve, AUC: 0.76 ± 0.05). Project-specific attributes—including the size of the proposing team, the time elapsed since their initial submission, measures of gender equity within the team, and whether the proposal was standalone or internally generated—also proved influential (AUC: 0.83 ± 0.04). Furthermore, reputational factors, such as the H-indices of the authors submitting the proposal, the overall size of the relevant research community, the proposers’ centrality within the NHGRI network, and the breadth of community support, showed strong predictive power (AUC: 0.87 ± 0.04). Linguistic elements of the proposals themselves, like the strength of argumentation and the degree of repetitiveness, also contributed to the model’s accuracy (AUC: 0.85 ± 0.04).</p>
<p>Crucially, when all these feature sets were combined, the model achieved a high level of performance in predicting funding decisions (AUC: 0.94 ± 0.03). This indicates that a multifaceted approach, considering biological, project-related, reputational, and linguistic factors, is necessary to understand the complexities of such funding allocations.</p>
</section>
<section id="interpreting-funding-decisions-the-matthew-effect" class="level2" data-number="18.11">
<h2 data-number="18.11" class="anchored" data-anchor-id="interpreting-funding-decisions-the-matthew-effect"><span class="header-section-number">18.11</span> Interpreting Funding Decisions: The Matthew Effect</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_15.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 15</figcaption>
</figure>
</div>
<p>To delve deeper into the factors driving funding decisions, researchers employed feature interpretability techniques. These methods help to elucidate how individual features within the computational model influence the predicted outcome, thereby identifying characteristics associated with a higher or lower probability of receiving funding. One significant observation from this analysis is the presence of a ‘Matthew Effect’, a phenomenon where initial advantages tend to accumulate further advantages.</p>
<p>Specifically, the analysis revealed that proposals submitted by authors with a higher maximum H-index were more likely to secure funding. Similarly, organisms supported by a larger, more established research community also had a greater chance of being selected for sequencing. This pattern aligns with the strategic objectives of funding agencies, which often prioritise projects perceived as having a higher potential for significant downstream impact and eventual clinical applications.</p>
</section>
<section id="broader-applications-from-archives-to-knowledge" class="level2" data-number="18.12">
<h2 data-number="18.12" class="anchored" data-anchor-id="broader-applications-from-archives-to-knowledge"><span class="header-section-number">18.12</span> Broader Applications: From Archives to Knowledge</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_16.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 16</figcaption>
</figure>
</div>
<p>The methodologies and insights gained from the NHGRI archive project extend to a broader vision of leveraging born-physical archives through computational analysis. This specific study serves as a compelling example of how such historical data, when processed with advanced tools, can yield valuable knowledge. The research consortium collaborates with various partners and utilises diverse data sources beyond the NHGRI, including federal court records from the United States and seismological data (seismograms) from the <em>EarthScope Consortium</em>.</p>
<p>A generalised workflow underpins these efforts. It begins with the acquisition of data and metadata from these archives. Subsequently, a sophisticated knowledge creation pipeline applies a series of computational processes. These include:</p>
<ul>
<li><p>page stream segmentation to delineate document structures</p></li>
<li><p>handwriting extraction</p></li>
<li><p>entity disambiguation to resolve identities</p></li>
<li><p>layout modelling to understand document formats</p></li>
<li><p>document categorisation</p></li>
<li><p>further entity recognition</p></li>
<li><p>redaction of personal information for privacy</p></li>
<li><p>modelling of decisions or processes captured in the records</p></li>
</ul>
<p>The ultimate aim of this comprehensive approach is to transform raw archival data into actionable insights that can address pressing scientific questions, inform policy-making, and significantly enhance the accessibility of these rich historical resources. This transformation relies on robust algorithms and a well-developed cyberinfrastructure.</p>
</section>
<section id="preservation-collaboration-and-nhgris-significance" class="level2" data-number="18.13">
<h2 data-number="18.13" class="anchored" data-anchor-id="preservation-collaboration-and-nhgris-significance"><span class="header-section-number">18.13</span> Preservation, Collaboration, and NHGRI’s Significance</h2>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="images/ai-nepi_020_slide_17.jpg" class="img-fluid figure-img"></p>
<figcaption>Slide 17</figcaption>
</figure>
</div>
<p>A significant challenge remains in the preservation of born-physical data, much of which currently resides in vulnerable conditions, such as shipping containers susceptible to damage and neglect. The imperative to safeguard these invaluable historical records for future scholarly and scientific inquiry cannot be overstated. Recognising this, the ‘Born Physical, Studied Digitally’ consortium actively seeks collaboration, inviting testers, partners, and users to engage with their tools and methodologies. This initiative receives support from prominent organisations including the National Institutes of Health (specifically NHGRI), <em>NVIDIA</em>, and the National Science Foundation (NSF).</p>
<p>The speaker also highlighted a pertinent contemporary issue: recent proposals within the United States to dissolve the NHGRI. This underscores the critical need to appreciate the agency’s historical contributions. Evidence suggests NHGRI stands as one of the most innovative funding agencies in the annals of science. Consequently, the rich data contained within its archives promise to unlock answers to numerous significant scientific questions, reinforcing the importance of its continued study and preservation.</p>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="./chapter_ai-nepi_019.html" class="pagination-link" aria-label="Modelling Context and its Interplay in Language Variation and Change: A Pilot Study on the Chemical Revolution">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text"><span class="chapter-number">17</span>&nbsp; <span class="chapter-title">Modelling Context and its Interplay in Language Variation and Change: A Pilot Study on the Chemical Revolution</span></span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="./chapter_ai-nepi_021.html" class="pagination-link" aria-label="From Source to Structure: Extracting Knowledge Graphs with LLMs">
        <span class="nav-page-text"><span class="chapter-number">19</span>&nbsp; <span class="chapter-title">From Source to Structure: Extracting Knowledge Graphs with LLMs</span></span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->




</body></html>